src/components/manage/Sharing/Sharing.jsx

```diff
@@ -412,10 +412,17 @@ class SharingComponent extends Component {
                           )}
                           {typeof entry.roles[role.id] === 'boolean' && (
                             <Checkbox
+                              id={`make-logged-in-users-as${role.id}${
+                                entry.roles[role.id] ? '-checked' : ''
+                              }`}
+                              name={`make-logged-in-users-as${role.id}${
+                                entry.roles[role.id] ? '-checked' : ''
+                              }`}
```

Comment:
Don't use "logged-in-users" as a fixed id/name; instead, use "role.id" to identify them correctly